suspicious-funding-ts/src/agent.spec.ts

```diff
@@ -0,0 +1,136 @@
+import {
+  FindingType,
+  FindingSeverity,
+  Finding,
+  HandleTransaction,
+  createTransactionEvent,
+  ethers,
+  Initialize,
+} from "forta-agent";
```

Comment:
`FindingType`, `FindingSeverity`, `Finding`, `createTransactionEvent`, and `ethers` don't seem to be in use.

Reply:
Fixed 👍